### DISCUSSION TITLE
Dan Abramov - React, Bluesky

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#274] [Announcements]
- https://twitter.com/dan_abramov2
- https://danabra.mov/
- https://overreacted.io/

### Blue Sky

- [x] What is blue sky? What do you think makes it different than other federated social media?
- [x] From what I knwo your time at facebook you mostly used react on the web. At bluesky you've been working on a react native app? What has the experience been like? 
- [x] How significant do you think react-strict-dom is?

### React

- [x] How does having many of the core React team members no longer working at Meta impact React's trajectory?
- [x] What do RSCs represent to you on react's trajectory? 
- [x] You've gone through a lot of public thinking on how to explain RSC. Why was it hard to explain them and what explanation did you land on?
- [x] The react compiler is a really interesting project. How do you think it will change the perception of writing react code? Will I ever need to use react hooks again?
- [ ] ~The `use` function is interesting what does it do? What type of code will change because of it?~
- [ ] ~What's your prefered way to write stlyes in react? stylex?~
- [ ] ~What are your thoughts on newer frontend framework like solidj qwik hono?~

### Future

- [ ] ~What upcoming features are you most excited for that are coming for react?~
- [x] What's the future of react?

**Comment by hipstersmoothie** [comment]
https://parceljs.org/

**Comment by just-be-dev** [comment]
https://www.holloway.com/
